### PR TITLE
#32377 added rules to make the code sniffer more compliant with ZF1.1…

### PR DESCRIPTION
**Pull request by d-heller**
…2 and the Sentact code standards

### CODE REVIEW

File: PHP/CodeSniffer/Standards/Sentact/ruleset.xml [hunk 1]
```diff
@@ -2,10 +2,19 @@
 <ruleset name="Sentact">
     <description>The Sentact coding standard.</description>
 
-    <rule ref="PSR2"/>
+    <rule ref="PSR2">
+        <exclude name="PSR1.Classes.ClassDeclaration.MissingNamespace"/>
```
Comment (dmertl) [inline PHP/CodeSniffer/Standards/Sentact/ruleset.xml:6]: Doesn't this remove the rule everywhere? We should have namespaces in normal files.

File: PHP/CodeSniffer/Standards/Sentact/ruleset.xml [hunk 2]
```diff
@@ -2,10 +2,19 @@
...
+        <exclude name="Squiz.Classes.ValidClassName.NotCamelCaps"/>
+    </rule>
 
     <!-- Proper docblock formatting -->
     <rule ref="Generic.Commenting.DocComment"/>
+    <rule ref="Generic.Commenting.DocComment.ShortNotCapital">
```
Comment (dmertl) [inline PHP/CodeSniffer/Standards/Sentact/ruleset.xml:12]: What are these for?

Comment (d-heller) [reply PHP/CodeSniffer/Standards/Sentact/ruleset.xml:12]: When you add descriptive info to a doc comment like outlining data formats, etc. the code sniffer will complain that something like 

`array(123)` will not start with an upper case

File: PHP/CodeSniffer/Standards/Sentact/ruleset.xml (hunk 1 above)
Comment (d-heller) [reply PHP/CodeSniffer/Standards/Sentact/ruleset.xml:6]: it does. If you see an option to add criteria to this that would be better. Alternatively we just need to live with the errors or remove the error output in general

Comment (dmertl) [reply PHP/CodeSniffer/Standards/Sentact/ruleset.xml:6]: Hmm. We definitely want namespaces in other files. I think it should be pretty hard to forget namespace if you're including the file anywhere else. Let's try it out and see how things go.

File: PHP/CodeSniffer/Standards/Sentact/ruleset.xml (hunk 2 above)
Comment (dmertl) [reply PHP/CodeSniffer/Standards/Sentact/ruleset.xml:12]: Ah, yeah. It's trying to encourage full sentences. I'm fine with taking that out.

**Pull request merged**